src/data/real/sqrt.lean

```diff
@@ -196,6 +211,26 @@ begin
   { exact le_trans hx (sqrt_nonneg y) }
 end
 
+lemma lt_sqrt (hx : 0 ≤ x) (hy : 0 ≤ y) : x < sqrt y ↔ x ^ 2 < y :=
+by rw [mul_self_lt_mul_self_iff hx (sqrt_nonneg y), pow_two, mul_self_sqrt hy]
+
+lemma lt_sqrt_of_sqr_lt (h : x^2 < y) : x < sqrt y :=
+begin
+  by_contra hnot,
+  rw [le_antisymm (le_sqrt_of_sqr_le _) (not_lt.mp hnot), sqr_sqrt] at h,
+  exacts [h.false, (lt_of_le_of_lt (pow_two_nonneg _) h).le, h.le],
+end
+
+lemma sqr_le' (h : x^2 ≤ y) : -sqrt y ≤ x ∧ x ≤ sqrt y :=
```

Comment:
I am still unsure of how to name this result. The only thing that comes to mind is `neg_sqrt_le_and_le_sqrt_of_sqr_le`, which is far too cumbersome.

Reply:
Possibly. It's certainly cleaner.